scripts/tag_grants_mesh_pipeline.sh

```diff
@@ -0,0 +1,7 @@
+python /pipelines/mesh/task_tagger.py \ 
```

Comment:
is this script supposed to be here?

Reply:
Lol, I committed this file by accident, ignore it, will delete.

Reply:
Done